Scrips.BaseDbContext/Scrips.BaseDbContext/ConnectionString.cs

```diff
@@ -0,0 +1,14 @@
+﻿using Microsoft.Extensions.Configuration;
+
+namespace Scrips.BaseDbContext
+{
+    public sealed class ConnectionString
+    {
+        public ConnectionString(IConfiguration configuration)
+        {
+            DefaultConnection = configuration.GetConnectionString("SchedulingDb");
```

Comment:
I think "schedulingDb" here by mistake, do you mean LoggingDb?

Reply:
Class is no longer in use so i removed it.